.github/workflows/build-gems.yml

```diff
@@ -7,18 +7,29 @@ on:
     branches: ["main", "cross-gem/*"]
 
 jobs:
+  ci-data:
+    runs-on: ubuntu-latest
+    outputs:
+      result: ${{ steps.fetch.outputs.result }}
+    steps:
+      - id: fetch
+        uses: oxidize-rb/actions/fetch-ci-data@main
+        with:
+          supported-ruby-platforms: |
+            exclude: [arm-linux] # no cranelift support yet
+          stable-ruby-versions: |
+            exclude: [head]
+
   native:
     name: Build native gems
+    needs: ci-data
     runs-on: ubuntu-latest
     strategy:
       fail-fast: false
       matrix:
-        ruby-platform:
-          - "aarch64-linux"
-          - "arm64-darwin"
-          - "x86_64-darwin"
-          - "x86_64-linux"
-          - "x86_64-linux-musl"
+        ruby-platform: ${{ fromJSON(needs.ci-data.outputs.result).supported-ruby-platforms }}
+    env:
+      CROSS_PLATFORMS: ${{ matrix.ruby-platform }}
```

Comment:
What read this? Couldn't find any reference to it in the gems (`rg CROSS_PLATFORMS $(bundle show --paths)`) nor the actions.

Reply:
Ahh, it's an old diff that is not needed anymore. Will delete.